### DISCUSSION TITLE
User status bug

### DISCUSSION THREAD

**Discussion by kahy9** [community/community#5868] [Other Feature Feedback, Questions, & Ideas]
When the user has set their status to `busy`, the user's status can cover up and mix with other content on the page.

![image](https://user-images.githubusercontent.com/48121432/135159562-ddbcfae7-bd02-4f02-919f-0efe8a325cb1.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
